contracts/Neumark.sol

```diff
@@ -125,6 +134,8 @@ contract Neumark is
         uint amount
     )
         internal
+        acceptAgreement(owner)
+        acceptAgreement(spender)
```

Comment:
we for sure should accept for `owner`. I would refrain from accepting in the name of `spender` as spender may not have a balance >0  as a result. he may be a broker.
I think we should ask Jacek. it may be we want also to include brokers in here.

Reply:
True. This may be redundant. The allowance case has not been discussed with Jacek.